linkerd/system/Cargo.toml

```diff
@@ -14,4 +14,4 @@ tracing = "0.1"
 
 [target.'cfg(target_os = "linux")'.dependencies]
 libc = "0.2"
-procinfo = "0.4"
+procfs = "0.15.1"
```

Comment:
I think its better to keep in all the features as is as few more features are yet to be implemented in procfs(not all proc features are yet available in this crate). This way in case we need anything more procfs features then we will have them with us

Reply:
Not sure I agree, we really only need this crate to get proc stats. To me, it makes more sense to have less transitive dependencies.